superdesk/media/renditions.py

```diff
@@ -392,6 +392,7 @@ def get_rendition_file_name(rendition):
     :param rendition:
     :return:
     """
-    ext = os.path.splitext(rendition.get('media'))[-1]
-    return rendition.get('media').replace('/', '-') + (
+    media_id = str(rendition.get('media'))
```

Comment:
I believe that rendition always has a `media`, but in theory, `media_id` might be `"None"`(string) here.

Reply:
yep I think so to, so will make it explicit not using `.get()`